### PR TITLE
Fix/import from git spec fail with non master default branch

### PR DESCRIPTION
**Pull request by lacostej**
On my computer, tests are failing as my default git branch is `main`.

### CODE REVIEW

**Comment by lacostej** [comment]
Looks like not everything is stable on the other platforms. I'll keep the master `branch` name for now.

File: fastlane/spec/actions_specs/import_from_git_spec.rb [hunk 1]
```diff
@@ -24,7 +24,11 @@
           source_directory_path = Dir.mktmpdir("fl_spec_import_from_git_source")
 
           Dir.chdir(source_directory_path) do
-            `git init -b master`
+            if Gem::Version.new(`git --version`.strip.split(' ').last) >= Gem::Version.new("2.28.0")
```
Comment (revolter) [inline fastlane/spec/actions_specs/import_from_git_spec.rb:27]: `git --version` outputs `git version 2.32.0 (Apple Git-132)` on my machine, so I think that this check would be incorrect.

Isn't there any other way to retrieve only the version, without needing to parse it?

**Review by revolter — CHANGES_REQUESTED** [review]

File: fastlane/spec/actions_specs/import_from_git_spec.rb (hunk 1 above)
Comment (revolter) [reply fastlane/spec/actions_specs/import_from_git_spec.rb:27]: [Apparently not](https://git-scm.com/docs/git-version) :(

Comment (revolter) [reply fastlane/spec/actions_specs/import_from_git_spec.rb:27]: Maybe using https://regex101.com/r/fImCOu/1 would be a better idea?

Comment (lacostej) [reply fastlane/spec/actions_specs/import_from_git_spec.rb]: @revolter yes, better. I even made most groups optional to parse a potential "3" version.

Comment (revolter) [reply fastlane/spec/actions_specs/import_from_git_spec.rb]: I didn't do that so as not to match the `132` in my example, but I guess that `.match()` only returns the first match, so it's all good 👍🏻

**Review by revolter — APPROVED** [review]
Thanks for the fix 💪🏻

**Pull request merged**